src/node/coinstats.cpp

```diff
@@ -25,6 +26,18 @@ uint64_t GetBogoSize(const CScript& scriptPubKey)
            scriptPubKey.size() /* scriptPubKey */;
 }
 
+static bool CanUseIndex(CHashWriter& ss)
+{
+    return false;
```

Comment:
5479000: this seems overly abstract, maybe just check the hash type in `GetUTXOStats`? Unless it makes more sense once MuHash is added.

Reply:
Done